src/components/common/CreateColonyWizard/validation.ts

```diff
@@ -42,9 +46,10 @@ const { formatMessage } = intl({
 });
 
 export const colonyNameValidationSchema = object({
-  displayName: string().required(
-    formatMessage({ id: 'error.colonyNameRequired' }),
-  ),
+  displayName: string()
+    .trim()
+    .max(MAX_DISPLAY_NAME)
```

Comment:
We need to pass a custom message as the second param, since the automatically generated message uses the field's key in the validation schema (`displayName`), which is ugly.

https://github.com/jquense/yup#stringmaxlimit-number--ref-message-string--function-schema

Reply:
I haven't done this, as I implemented your other suggestion of setting the `maxLength` of the Input to have consistency across the app. As this is the case, I have just removed this validation as this change makes it obsolete.